opsdroid/database/mongo/__init__.py

```diff
@@ -62,3 +62,13 @@ def __init__(self, config, opsdroid=None):
         return await self.database[key].find_one(
             {"$query": {}, "$orderby": {"$natural": -1}}
         )
+
+    async def delete(self, key):
+        """Delete a document from the database (key).
+
+        Args:
+            key (str): the key is the database name.
+
+        """
+        logging.debug("Deleting %s from mongo", key)
```

Comment:
Strange that this file isn't using the `_LOGGER` pattern. In most files we have `_LOGGER = logging.getLogger(__name__)` somewhere at the top and then would use `_LOGGER.debug` here.

I appreciate this isn't your problem but would you mind changing this file to follow the pattern? See the redis database for reference.

Reply:
Made the mentioned changes.